hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -189,20 +189,18 @@ public long contractCreationLimit() {
     public long lazyCreationCostInGas(@NonNull final Address recipient) {
         final var payerId = context.payer();
         // Calculate gas for a CryptoCreateTransactionBody with an alias address
-        final var createFee = gasCalculator.topLevelGasRequirement(
```

Comment:
Do we need `topLevelGasRequirement` anymore?

Reply:
Hmm maybe not, but can't hurt too much I suppose.